tests/conftest.py

```diff
@@ -165,3 +167,19 @@ def garbage(self, restrictions) -> Response:  # pylint: disable=unused-argument
             return Response("x")
 
     return XResource
+
+
+@pytest.fixture()
+def sample_resource_without_authorization() -> Type[Resource]:
```

Comment:
This can be named simply `sample_resource` if we have version `with_authorization` above 😉